### PR TITLE
update ubereats-connect.md

### PR DESCRIPTION
**Pull request by janainawittner**
@atorin, For your information, I did not make comments already reported for Deliveroo....

Working with Pull Requests when I have no intention of writing / taking ownership of this documentation does not work. So, I am looking for another way to give feedback in githut. 

Using issues does not work either unfortunately as we cannot easily make notes when the page is in markdown. We looked at it with Antoine :-(

Here is what I have done instead. I am using pulll request but my text input will merely be : // JW 
I will do this when I have a comment to make on the text. This is the only way I found to select the text my comment is related to and give you some context. 

Let's see if this works....

### CODE REVIEW

File: content/apps/uber-eats/en/connect-hubrise.md [hunk 1]
```diff
@@ -41,17 +41,17 @@ If you don't have a HubRise account, go to the [HubRise Signup Page](https://man
 You have now added Uber Eats to your HubRise apps.
 The next step is configure the connection.
 
-## Store ID
+## Store ID // JW
 
 ---
 
-**IMPORTANT NOTE:** In order to connect your Uber Eats restaurant to HubRise, you need to provide a valid store ID.
+**IMPORTANT NOTE:** In order to connect your Uber Eats restaurant to HubRise, you need to provide a valid store ID. // JW
```
Comment (janainawittner) [inline content/apps/uber-eats/en/connect-hubrise.md:48]: Here you say "you need"....
The restaurant owner will not provide this. HubRise will get it from the Uber Eats technical team thanks to the email he/she will have sent us to innitiate the request.

File: content/apps/uber-eats/en/connect-hubrise.md [hunk 2]
```diff
@@ -24,12 +24,12 @@ We will then contact Uber Eats on your behalf to establish the connection and ge
...
 ---
 
-**IMPORTANT NOTE:** Uber Eats Bridge connects to HubRise at the location level.
+**IMPORTANT NOTE:** Uber Eats Bridge connects to HubRise at the location level. // JW
 
 ---
 
 To connect Uber Eats to HubRise, you first need to log in to your HubRise account.
-If you don't have a HubRise account, go to the [HubRise Signup Page](https://manager.hubrise.com/signup) and follow the instructions.
+If you don't have a HubRise account, go to the [HubRise Signup Page](https://manager.hubrise.com/signup) and follow the instructions. // JW
```
Comment (janainawittner) [inline content/apps/uber-eats/en/connect-hubrise.md:32]: The item we are liking to should be at the very end f the sentence: 
https://www.hubrise.com/contributing/style-guide#cross-references

File: content/apps/uber-eats/en/connect-hubrise.md [hunk 3]
```diff
@@ -41,17 +41,17 @@ If you don't have a HubRise account, go to the [HubRise Signup Page](https://man
...
 
 ---
 
-**IMPORTANT NOTE:** In order to connect your Uber Eats restaurant to HubRise, you need to provide a valid store ID.
+**IMPORTANT NOTE:** In order to connect your Uber Eats restaurant to HubRise, you need to provide a valid store ID. // JW
 
 ---
 
-In the first configuration page, you will need to enter the store ID that we will provide you after contacting Uber Eats.
+In the first configuration page, you will need to enter the store ID that we will provide you after contacting Uber Eats. // JW
```
Comment (janainawittner) [inline content/apps/uber-eats/en/connect-hubrise.md:52]: I think it's easier if we include this UUID in the Bridge.
I would explain how this is done but we will put it. It's easier than writing to the client and explaining how to do it :-)

File: content/apps/uber-eats/en/connect-hubrise.md [hunk 4]
```diff
@@ -41,17 +41,17 @@ If you don't have a HubRise account, go to the [HubRise Signup Page](https://man
 You have now added Uber Eats to your HubRise apps.
 The next step is configure the connection.
 
-## Store ID
+## Store ID // JW
```
[the remark this answers is not in the document]
Comment (atorin) [reply content/apps/uber-eats/en/connect-hubrise.md]: I will change ID to UUID. 

The UUID page is separate from the configuration page, if I understand correctly your question. 
Can we establish a connection without it?

File: content/apps/uber-eats/en/connect-hubrise.md (hunk 1 above)
Comment (atorin) [reply content/apps/uber-eats/en/connect-hubrise.md]: You're right, it was a poor choice of words.

File: content/apps/uber-eats/en/connect-hubrise.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (atorin) [reply content/apps/uber-eats/en/connect-hubrise.md]: @antmonnier We will also need to change the UI (it currently says "Store ID"). I'll make a note for this.

File: content/apps/uber-eats/en/connect-hubrise.md (hunk 3 above)
Comment (atorin) [reply content/apps/uber-eats/en/connect-hubrise.md]: OK, I rephrased the paragraph. Let me know if this is better.

**Pull request merged**